lightning/src/util/test_utils.rs

```diff
@@ -300,6 +302,22 @@ impl TestLogger {
 		let log_entries = self.lines.lock().unwrap();
 		assert_eq!(log_entries.get(&(module, line)), Some(&count));
 	}
+
+	pub fn assert_log_contains(&self, module: String, line: String, count: usize) {
```

Comment:
Good catch. Using `find` here is my mistake. It was accidentally working because so far every assertion works with `count = 1` . I should use `filter` and `sum` instead.